components/operator/api/v1alpha1/serverless_types.go

```diff
@@ -26,6 +27,14 @@ type DockerRegistry struct {
 	EnableInternal *bool `json:"enableInternal,omitempty"`
 	// Secret used for configuration of the Docker registry
 	SecretName *string `json:"secretName,omitempty"`
+	// PV configuration
+
+	PV *PersistanceVolume `json:"pv,omitempty"`
+}
+
+type PersistanceVolume struct {
+	// Size of the Docker registry PV
+	Size *resource.Quantity `json:"size,omitempty"`
```

Comment:
Why we have `ommitempty`? Is this value optional? Why? Does `PV` with empty `Size` make sense?

Reply:
thx, fixed